integration_test/cloudbuild-integration-test.yaml

```diff
@@ -143,17 +155,46 @@
   - '-Dexec.args=-c IMPORTADAPTER@${_ADAPTER_RUN_STEP}:${_ADAPTER_PORT} -m SOPInstanceUID=${_REPLACED_UID} --dest STORESCP'
   dir: "dcm4che/dcm4che-tool/dcm4che-tool-movescu"
   waitFor:
-  - run-commitment-scu
+  - run-store-scu
   - wait-for-storescp
 
+# Stgcmtscu  waits for n-event-report after n-action request returns success,
+# if it doesn't get one within cloud build step timeout of 30 seconds, step fails.
+# Run it after other tools to avoid timeout due to parallel execution.
+- name: ${_MAVEN_IMAGE}
```

Comment:
This is taking more than 30s to complete when there are other requests?

Reply:
Each cloud build step adds about 5s delay just for container initialization/teardown. So it's not adapter itself that takes 30s to process few minimalistic queries, commitment step failed because it didn't initialize in time.